modules/cli/src/main/kotlin/datamaintain/cli/update/db/UpdateDb.kt

```diff
@@ -72,5 +76,6 @@ class UpdateDb(runner: (DatamaintainConfig) -> Unit = ::defaultUpdateDbRunner) :
         }
         checkRules?.let { props.put(CoreConfigKey.CHECK_RULES.key, it.joinToString(",")) }
         allowAutoOverride?.let { props.put(CoreConfigKey.PRUNE_OVERRIDE_UPDATED_SCRIPTS.key, it.toString()) }
+        flags?.let { props.put(CoreConfigKey.FLAGS.key, it.joinToString(",")) }
```

Comment:
Since such a join is also done for check rules (and I guess it'll be done if there are other list options), it could be useful to create an extension of List in this file that does this join. That would prevent incoherence between the behaviours for multiple lists.